### PR TITLE
Remove legacy rsk-utils package

### PR DESCRIPTION
**Pull request by jessgusclark**
Remove the legacy rsk-utils package and instead copy and paste in the methods.

- the file is copy/pasted exactly with only linting changes to adhere to the style of the repo.
- this is a temporary step to get this working for the release.
- when we have more time, we may create a new reusable package for these methods to provide them a home.

### CODE REVIEW

**Review by Freshenext — APPROVED** [review]
Code looks good 😁

File: src/lib/addresses.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+ */
+
+/**
+ * @description Check if a string is an address
+ * @param {String} address
+ * @returns {Boolean}
+ */
+export function isAddress(address) {
+  return /^(0x)?[0-9a-fA-F]{40}$/.test(address)
+}
```
Comment (shenshin) [inline src/lib/addresses.js:12]: viem already has methods for this
[address utilities](https://viem.sh/docs/ethers-migration#address-utilities)

**Review by shenshin — APPROVED** [review]
Now looks perfect

**Comment by jessgusclark** [comment]
Awesome, thanks guys!

**Pull request merged**